scripts/west_commands/runners/nrfjprog.py

```diff
@@ -103,8 +113,13 @@ def do_run(self, command, **kwargs):
                              format(self.hex_) +
                              'Try enabling CONFIG_BUILD_OUTPUT_HEX.')
 
-        program_cmd = ['nrfjprog', '--program', self.hex_, '-f', self.family,
-                       '--snr', board_snr]
+        if self.family != 'NRF53':
+            program_cmd = ['nrfjprog', '--program', self.hex_, '-f', self.family,
+                '--snr', board_snr]
+        else:
+            program_cmd = ['nrfjprog', '--program', self.hex_, '-f', self.family,
+                '--snr', board_snr] + self.tool_opt
```

Comment:
tool_opt should be available for all families, not just NRF53.

Reply:
OK, I am making this available for all platforms